### DISCUSSION TITLE
Unguided Campaign Discussion

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#185] [Show and tell]
And we're back. I thought I would move this to here from the [Issue ](https://github.com/swfans/swars/issues/58) as it works better in this forum-y format and won't languish on page 97.

Anyway, I did some more work on my Trash Church adaptation, but got a bit bored, so went back to something else I worked on ages ago and forgot about. Now, there are two versions of "Guns Guns Guns" that shipped with the game, an old format level, and a newer one. The old one worked fairly well, but had a load of clearly broken stuff caused by the map being redesigned at some point, so I decided to have a go fixing it up. This is obviously lower hanging fruit than making a whole new level.

There were no objectives, so I made my own based on what behaviour was defined in the level. This is actually a pretty heavily scripted mission, so it's easy to figure out what was intended. A lot of it only made sense if objectives were pointing to very specific places, so in they went. I tried to use the fixed GAME_OBJ_ALL_G_USE_V objective @mefistotelis worked on, but it still doesn't work right, it correctly highlights the intended vehicle, but I couldn't get it to complete the objective when boarded, even when using the amount argument. As such I've just made the vehicle boarding objectives a Go to location objective for now instead. 

I've moved the soldier who was set to blow up the tower to start hidden in one of the bunkers instead of stuck in the wall, as well as his armoury of weaponry, and redirected all the pointers that were waiting for the player to go to the wall area to trigger here instead. He's still a bit off, if he gets into the main building he ends up getting just stuck in the wall instead of blowing up the reactor thingy, so I guess I would have to use a "Use Weapon" objective to force him to do it properly. Interestingly he keeps trying to walk through the wall where he used to start, so there seem to be pathfinding issues in the actual map for this city from when they changed it (some of the agents rarely do this too).

Anyway, the objective is to gather your punks, then rearm from the bunker, blow up the reactor things in the base, and then retrieve the stole unguided bike from the top base. I was going to have a point to take the bike to, but ran out of mission objectives!

Downloads:

[Level File](https://filebin.net/0swkkbzr3fa5ihyr)
[New miss002.ini Campaign file](https://pastebin.com/1n5tJc3H)

Campaign changes:

I looked at the "new" version of the level, but while superficially it looks finished, it's brutally hard and full of unused functionality that makes no sense. To cut to the chase it's clearly _not_ finished or coherent at all. Therefore I propose to abandon it in favour of this old version, which works and is quite fun.

Secondly I also vote to remove the "Clean Up" protests level from the campaign. This is a really fantastic level concept, but I've already written about a million times how it just doesn't work, so it gets the chop.

I think we should take out the obviously broken levels so you can complete the campaign. The next up for scrutiny would be the two weird "communications" levels, but I think they can be salvaged by moving them to the Reykjavik map.

If the above seems ok to everyone, I will make a proper pull request, etc

**Comment by mefistotelis** [comment]
>  I tried to use the fixed GAME_OBJ_ALL_G_USE_V objective @mefistotelis worked on, but it still doesn't work right, it correctly highlights the intended vehicle, but I couldn't get it to complete the objective when boarded

Try `GAME_OBJ_MEM_G_USE_V` - this one does accepts `Amount`. the `ALL` version requires all non-dead group members to be in the vehicle.

Hm, also it looks like i incorrectly named `ObvStatu_COMPLETED` and `ObvStatu_FAILED`, they are switched; will have to fix that.

EDIT:
For how to move forward with the levels, I have no objections. I don't think I even played the pre-alpha unguided levels.

When creating a campaign from scratch, you get the story and the ideas for some core level concepts, you match these, add more level concepts in-between, and then start with the implementation in map editor.
We are not creating from scratch, so we've started by examining and salvaging pre-existing levels. Removing the stuff which didn't worked is a required part of this step.

We should probably start gathering ideas for next steps, though. Like the story, and the concepts for missing levels.
I think we did discussed some already? not sure where though. Maybe we should maintain a list.

What comes to my mind right now is a concept of a level where you don't have weapons and have no ability to acquire weapons, but your team is modded, so you can use self-destruct:

1. self-destruct vs. a building
With modded agent who is able to self-destruct, you need to destroy a building; but just running there won't work, as you will be gunned down way before reaching the target. So you need to steal a car, and use it to reach the target, then self-destruct.
If that's not enough, the destroyed building may be a bank and next you need to pick up the suitcases dropped in the rubble.

Suicide bombing a bank could also be a separate mission.

2. self-destruct vs. crowd
This terroristic aspect is missing in the SW story, even though you could really already do it even in the original Syndicate. Here you may have only clone shield, or maybe you won't be perceived as a threat due to no weapons. Anyway you have to reach a crowd of civilians - maybe people listening to cultists - and self-destruct there to cause panic.. Then maybe some more Unguided will appear, with story justification that the civilians wake up against the church? I don't know, the rest is less important - the part to remember is the suicide bombing.
Though I wonder if the Unguided faction should really behave like that.. they are terroristic, but not sure if against civilians.. If this seem wrong, the mission could be instead to kill NPC suicide bombers.

**Reply by Moburma** [reply]
> > I tried to use the fixed GAME_OBJ_ALL_G_USE_V objective @mefistotelis worked on, but it still doesn't work right, it correctly highlights the intended vehicle, but I couldn't get it to complete the objective when boarded
> 
> Try `GAME_OBJ_MEM_G_USE_V` - this one does accepts `Amount`. the `ALL` version requires all non-dead group members to be in the vehicle.
> 
> Hm, also it looks like i incorrectly named `ObvStatu_COMPLETED` and `ObvStatu_FAILED`, they are switched; will have to fix that.

Ahh, you are correct as always. using MEM_G_USE_V does actually work, I stuck a new version of the campaign .ini above using this. However, GAME_OBJ_ALL_G_USE_V still does not seem to work. It identifies the vehicle, but even with all units in the group in the car, it shows as not complete. I believe neither of these objectives are used in the final game. MEM_G_USE_V is interesting as it shows a persuade style indicator from your unit to the target vehicle.

I've got a big response to the rest of your post, so will do that as a different reply.

**Comment by mefistotelis** [comment]
> However, GAME_OBJ_ALL_G_USE_V still does not seem to work.

I see two possible causes:
1. I misinterpreted how the game stores whether multiple people are in a vehicle. If this is the case, any `Amount` larger than 1 won't work, so it's easy to check.
2. There are additional things in the player group; since they never board the vehicle, it isn't possible to meet the objective. This could be checked by changing the group index.

**Comment by Moburma** [comment]
### Unguided Campaign Plot

I have to admit I don't have some inspired grand narrative for how this should play out. I think doing something different from the other campaigns would be interesting (i.e. not ending up on the moon), but at the same time we're limited with what tools we have. I also believe that we've already got the unguided ending in the game already. It's clear they changed some elements of the zealot campaign plot, probably due to the punk one being cancelled. That's why the ending there switches up to suddenly talking about Vissick having a new way forward for mankind. 

That would be the obvious ending anyway - a "good" ending where he triumphs and the oppressive church AND corporations are beaten. How we get there is another question...

Anyway, here are my notes on Unguided plot stuff that is present in the final game as a basic scaffold of what it seems was intended in the order it comes up:

- Unguided uprising is said to start with Vissick sending some kind of "seminal message" to the world, urging the unguided to rise up.
This specifically spells out the plan that the Church of the New Epoch will bring about global annihilation. Would make most sense as the briefing to mission 1 [source: Syndicate campaign briefings]

- First mission - Zealots release Harbinger, Unguided kill them after this event, underlines unguided and zealots are not aligned, despite punks needing this event to begin to truly break free

- Mission Guns Guns Guns - attempt to get hold of better weaponry? Not clear what the intention of these was, the "new" one you get a sniper rifle at the end, the "old" one is a kind of military arms depot with lots of munitions

- Heavy unguided attacks on zealots in Hong Kong and Rome [source: Zealot campaign briefings]

- Mission Rescue - rescuing a scientist for some reason - can't be Drennan. A defector?

- Mission Massive Assault - Some kind of large scale fight against Eurocorp and/or zealots, presumably making military gains and becoming a fighting force

- Ferocity - assassination of prof Drennan - what do Unguided gain from this?

- Talks Ambush - Presumably result of above, Eurocorp want fake peace talk after the blow of losing Drennan

- Mission Communications -  Trying to cut Eurocorp comms, or trying to establish their own?

- In the zealot campaign, the unguided are hunted down to Johannesburg. It's pretty clear that in fact Vissick was intended to be killed here, but in the final campaign it's some kind of second in command character. Either way, could be a good potential survival mission

- Vissick and the Unguided gather in Sevastopol and Delhi. Deel Gonda is in Delhi. They seem to be working East from here to Tibet, but it's never made clear why.

- They also attack Bahrain and attempt to destroy the Bahrain AI as this controls most of the middle East for Eurocorp

- They then plan to escape to Lake Margui Caka on the Tibetan plateau. For some reason Vissick believes they will survive the apocalypse here

- It's not clear what was intended from here, if they just went via Columbo to the moon like the other factions, or what

From ChatGPT on why the Tibetan Plateau is unique in the world:

High Altitude: The Tibetan Plateau is the highest and largest plateau in the world. Its average elevation is over 4,500 meters (14,800 feet) above sea level, which is significantly higher than most other plateaus. The plateau includes the Himalayan mountain range, which is home to Mount Everest, the highest peak on Earth, at 8,848 meters (29,029 feet).

**Reply by Moburma** [reply]
More ChatGPT ruminations trying to work out why they chose this place as the destination for salvation:

```
If a hypothetical superweapon could create a hole in the ionosphere to direct cosmic rays at specific parts of the Earth, the Tibetan Plateau might offer some degree of protection, but not absolute safety. Here's why:

1. Altitude and Atmospheric Shielding:
The Tibetan Plateau is one of the highest regions on Earth, with an average elevation of about 4,500 meters (14,800 feet) above sea level. While this high altitude places people closer to cosmic rays, the remaining atmosphere above still provides significant shielding.
The Earth's atmosphere, even at higher altitudes, absorbs and scatters cosmic rays, reducing their intensity as they reach the surface. The higher you go, the thinner the atmosphere, and the less protection you have, but there would still be some level of protection on the Tibetan Plateau compared to areas directly under the hole in the ionosphere.
2. Geomagnetic Protection:
The Earth's magnetic field also plays a role in protecting against cosmic rays, particularly charged particles. The Tibetan Plateau is located at a mid-latitude, where geomagnetic protection is stronger compared to the poles. However, this protection is less effective against high-energy cosmic rays.
A breach in the ionosphere could potentially allow more cosmic rays to penetrate the atmosphere, but the geomagnetic field would still deflect some of these particles, especially near the equator, though Tibet is somewhat closer to this protection than polar regions.
3. Cosmic Ray Intensity:
The intensity and type of cosmic rays would matter. Low-energy particles are more likely to be absorbed by the remaining atmosphere, while high-energy particles could penetrate more deeply. The Tibetan Plateau's atmospheric thickness is still sufficient to offer some protection against low and moderate-energy cosmic rays.
4. Geographical Location:
The Tibetan Plateau is far from the equator, where cosmic rays are typically more intense due to the thinner atmosphere and weaker geomagnetic field. Being closer to the equator generally offers better protection, but the plateau's high elevation means it wouldn't be as well-protected as lower-altitude regions at the same latitude.
Conclusion:
Relative Safety: The Tibetan Plateau could provide some level of protection due to the remaining atmosphere above it, but it wouldn't be the safest location compared to lower altitudes, especially if the superweapon could focus cosmic rays precisely.
Better Alternatives: Areas with lower altitudes, closer to sea level, and within stronger geomagnetic field zones would offer better protection against cosmic rays. The Tibetan Plateau might be relatively safer compared to other high-altitude or polar regions, but it wouldn't be the ideal location to avoid the effects of such a superweapon.

```

I still don't get what they were going for. The geomagnetic protection angle is interesting but sounds pretty weak as something to hang a story around. This is looking at it from the perspective of their being something physically unique about the actual _location_, I guess the other way to go would be more basic - it's a remote place, perhaps there was some kind of forgotten top secret military base/bunker there? 

From a geographical point of view it seems it would actually be quite a rubbish place to try and survive from cosmic rays!

**Comment by Moburma** [comment]
### Game Characters

There aren't many actual characters in the game, but here are outlines of the few who could be relevant in an unguided campaign:

**Vissick** - Genius scientist who was part of the Eurocorp C3 team. Worked with the scientists who became the Nine and knows their plans and mentality. Lore states Tyrone Chodak (a future member of the Nine) was his nemesis, pushing to interface with the alien technology that in the end drove him mad. It's confirmed the player would "be" Vissick in the punk campaign. Figuring out what sort of character he would be is one of the biggest issues to solve.

**Tyrone Chodak** - Leader of the Nine/zealots, and the one who decided to interface with alien technology despite Vissick's protests. Wrote the Codex, the main work of alien technology and information that forms the basis of the Church philosophy. only mentioned once at all in the actual game. Could be a potential antagonist in briefings etc

**Mirabelle Lucy De Saxo** - A "special infiltration agent" for the Church of the New Epoch. She abducts Professor Brennan, but in fact turns out to be a double agent working for Eurocorp. She is a double agent working for the opposite side in both campaigns. Debateable what she would do in an Unguided campaign, but she is present in "Ferocity". The Eurocorp executive Anika Maritz is actually her in disguise.

**Deel Gonda** - "An ex-EuroCorp strategist", he is said to be aiding Vissick in the later stages of the campaign, particularly guiding their fighting in Delhi and Sevastopol. That he's a Eurocorp defector could be something that could be played on.

**Anika Maritz** - a UTOPIA Agent/Eurocorp executive, who briefs the player for most of the Eurocorp campaign. However, the London AI begins to suspect she may be a double agent due to her behaviour around the Drennan incident, and eventually she is killed by the player. It's revealed she is actually De Saxo, who in the Eurocorp campaign is a zealot agent.

**Rene Galact** - Character said to be leading Unguided rebellion against the COTNE in Johannesburg

**Chico Ramirez** - Character said to be leading Unguided rebellion against the COTNE in Buenos Aires

**Comment by Moburma** [comment]
### Are the Unguided actually morally good?

>Though I wonder if the Unguided faction should really behave like that.. they are terroristic, but not sure if against civilians.. 

I think this is one of the biggest issues to solve. The level "Clean Up" presents them as unambiguously good, as they are defending civilians protesting from Eurocorp agents. However, pretty much everywhere else in the game they are not morally righteous at all, seemingly with the attitude that the ends justify the means. They also regularly seem to slaughter civilians for no apparent reason (see Eurocorp mission 1). To use a kind of D&D trope, they seem to be a chaotic neutral faction. 

Amorality is also the order of the day in Syndicate Wars, so why would they be any different? I guess the big issue is Vissick, and his character. He's a scientist, so you would immediately expect  a cliched calm and rational stance from him. The zealot ending where he is poised to usher in a new era for mankind also affirms this. So how does he square this with working with people who seemingly gun down normal people for kicks? Maybe he doesn't care, he's too jaded by what he saw at Eurocorp? Or maybe there is constant conflict between him and his allies (maybe there is a coup attempt by other unguided internal factions)?

**Comment by Moburma** [comment]
### Mission Ideas

I have said this lots of times already, but I personally see the punk campaign as an opportunity to take some gameplay elements to their logical conclusions. Bullfrog were usually very good at this, but SW does this far less, as the ability for the player to bring whatever and whoever they like in the other campaigns leads to certain strategies/equipment inevitably dominating. In comparison, Magic Carpet and Dungeon Keeper made great hay from restricting what the player could have/do, leading to memorable and fun levels beyond the obvious.

> What comes to my mind right now is a concept of a level where you don't have weapons and have no ability to acquire weapons, but your team is modded, so you can use self-destruct:
> 
> 1. self-destruct vs. a building
>    With modded agent who is able to self-destruct, you need to destroy a building; but just running there won't work, as you will be gunned down way before reaching the target. So you need to steal a car, and use it to reach the target, then self-destruct.
>    If that's not enough, the destroyed building may be a bank and next you need to pick up the suitcases dropped in the rubble.
> 
> Suicide bombing a bank could also be a separate mission.
> 
> 2. self-destruct vs. crowd
>    This terroristic aspect is missing in the SW story, even though you could really already do it even in the original Syndicate. Here you may have only clone shield, or maybe you won't be perceived as a threat due to no weapons. Anyway you have to reach a crowd of civilians - maybe people listening to cultists - and self-destruct there to cause panic.. Then maybe some more Unguided will appear, with story justification that the civilians wake up against the church? I don't know, the rest is less important - the part to remember is the suicide bombing.
>    Though I wonder if the Unguided faction should really behave like that.. they are terroristic, but not sure if against civilians.. If this seem wrong, the mission could be instead to kill NPC suicide bombers.

This is really good and just what I mean!

Other ideas:

- A [Commandos](https://store.steampowered.com/app/6800/Commandos_Behind_Enemy_Lines/) style stealth mission. Use clone shield and maybe Displacertron to get through a level where the player is either unarmed or massively outgunned. Works well with early low level unguided who have very limited energy levels
- Tower defence style mission using stock pile of cerberus IFFs and Ion Mines, Razor/Explode Wire, etc. Player has to arm up and then waves of enemies attack. Between some waves you get more unguided reinforcements. The mission "Day Of The Damned" would be a good starting template for this, perhaps you have to defend Vissick
- Hell on two wheels - level where you have one unguided and have to assassinate lots of individual targets. Start off next to a motorbike and have to pick each one off, probably time limit for each one before they run away etc
- Base full of enemies, use psycho gas to make them fight each other - perhaps they are different Eurocorp factions uneasily reuniting after the UTOPIA outage made them realise they could actually go it alone if they dared

**Comment by Moburma** [comment]
### Stamina Design Decision Needed
Something that is apparent throughout all the pre-alpha Unguided missions is that the punks have very low stamina. This seems to be in line with their values as NPCs in the greater game. What this means in practice is they get exhausted very quickly, and so the player characters can't run about much before being so tired they are forced to walk. 

However, in the much later designed mission A Tale of Two Cities, the unguided there are in line with other late game characters, and can happily sprint all day long.

Therefore either:

- It was intended that the unguided were more vulnerable and thus had this limitation, but it was to reduce as the game goes on (and difficulty increases) to be fair
- It's a bug/oversight caused by the game changing drastically since the pre-alpha days when most unguided levels were made and then abandoned, and in fact they were not supposed to be restricted (or at least not as much)

It's not clear which is the case. I think it would be interesting to stick with the first interpretation as it does indeed emphasize the greater fragility and human nature of the unguided and forces a different play style. It's also worth considering that Agents and Zealots of course initially also DO have this restriction as well, but it quickly gets removed/reduced once the body mods start rolling in.

**However**, it's also frankly quite annoying. It quickly gets tiresome when having to cover larger distances, and losing dramatic firefights because your people were too tired to move is lame. So it needs to be thought about and implemented carefully, if it stays in. I quite like it on my stealth level if that goes in, but even then it needs to tinkered with to not be irritating. Also thought needs to be taken as to how it should improve throughout the campaign.

**Reply by mefistotelis** [reply]
I like having differences between factions - so if the stamina limit may be considered one of defining features of Unguided - that's good. We don't want all factions to play the same, and differ only in cosmetics. This means each should have a strongly visible weakness. The Zealots and Agents actually are too similar.

To factor in the stamina, most levels can be just designed so that you're facing small groups rather than getting whole level antagonized. For levels where low stamina becomes a problem, we could place mods on the level - this way the player may easily notice the difference after getting them. And that can be achieved either by pickup, or by visiting an implanting shop - more cyberpunky I guess.

Currently the only mods pickable from level are skin mods (WeaponType=31, Ammo=mod level), but we could make it support all mods. We should stop re-using weapon 31 for different stuff anyway.
And/or using commands - `TNG_CMD_GIVE_CYBMOD` or `TNG_CMD_SET_STAMINA` are possible to implement as well. (that brings a different discussion - whether stats should be set individually, or it's better to re-compute them based on person type and mods).

**Comment by Moburma** [comment]
###What should the Unguided background look like?

I've been doing a lot of thinking about how the campaign should work for a while. This is seemingly one of the simpler questions to be answered, but actually leads to a lot of sub-questions when you think about it.

The other campaigns work that you the player aren't actually anyone that important, you're kind of a mid-level cog in both organisations, just doing what you are told by stronger forces. So therefore it makes sense that would likely be the case here too. While some of the original press releases etc state the player would 'be' Vissick, it doesn't actually make sense in practice. He's likely coordinating disparate groups around the globe, possibly through trusted generals etc, but at the same time he starts seemingly from nothing. Also why would he be reading/receiving emails he himself wrote!

So my idea would be more the player would either be some kind of trusted general, or more likely just whoever knows where to look to see the latest messages on public terminals etc. Basically a member of an unguided 'cell'.

Anyway, to this end I got some AI slop generated to indicate the kind of thing I personally envisage:

https://ibb.co/Hfjd9hn0
https://ibb.co/ch5b7BLP
https://ibb.co/kYHrMCH

![](https://i.ibb.co/dwRgHmGS/20250512-0956-Neon-Street-Terminal-simple-compose-01jv1v7wccebstmc34qwndf8mn.png)

![](https://i.ibb.co/QvB81gdZ/20250512-1001-Neon-Cyberpunk-Street-simple-compose-01jv1vm36ffd4s8dej1tfsb6zy.png)

![](https://i.ibb.co/77YTbmY/terminal.webp)

You get the idea. It would be unguided jacking in on the street before moving to the target location and the mission. 

Against it would be the fact that the other missions are supposed to be an operative in an airship controlling people on the ground, hence the  viewpoint, but I don't really get how that would work plotwise anyway (who would be in the airship, Vissick?) so I gloss over this.
Thoughts?

**Reply by mefistotelis** [reply]
Well we definitely need horizontal background.

I was imagining something more like a hackers cellar, with lots of monitors and boxes with electronic junk (mods?) laying around. Dusty atmosphere, maybe a primitive raw steel operating table in the background. Either without windows or with them covered and  light going in only through holes.

Street terminal is an interesting setting as well, but we'd need a holo projector there.

**Reply by Moburma** [reply]
> Well we definitely need horizontal background.

Yeah, that's just where it defaulted to the horrible phone orientation, by the time I got it generating something like I wanted I had run out of prompts for the day! But it's more the concept than the details.

>I was imagining something more like a hackers cellar, with lots of monitors and boxes with electronic junk (mods?) laying around. Dusty atmosphere, maybe a primitive raw steel operating table in the background. Either without windows or with them covered and light going in only through holes.

Yeah, that's the other thing I thought of, but still need to nail down the question of WHO is it reading the emails?
One thing I wanted as well was to play up the whole 'bikers' angle too, ironically the game as-is doesn't actually do that very much. I think it's an exciting way to differentiate them from the more sober corporate and crazy religious zealot types. 

This is the other problem with who are the unguided, are they tech savvy or more just violent punks? Is there a clash between them and Vissick in that sense (as a scientist), or are they are already very much _cyber_ punks?

Anyway, I already wrote some example mission briefings, I will do more until I have some I am happy with, hopefully some of these issues will work out naturally.

**Reply by geist22** [reply]
Is there a possibility that the player isn't just someone controlling Vissick's minions at his behest, but rather one of the Unguided on the field doing the actual work themselves? In that sense you'd take the role of a different person for each mission, and the Unguided do have a free will after all, unlike Syndicate/Church agents that are basically mind-controlled puppets that only exist to serve their masters much like robots.

In the case of being someone controlling others like in the other campaigns, I'd imagine the player being some prominent cyberhacker. Someone who's not particularly loyal to Vissick, but shares common interests, and does some odd jobs for him that fit his agenda (and may quit or be betrayed himself at any time).

The Unguided do not appear to be unified in the sense that you might call them a real faction. Some of them might align with Vissick, while others (i.e. the ones massacring civilians) might just be local gangs doing their own thing, and possibly hostile towards other Unguided. But since the latter undermine EuroCorp's rule and waste their time and resources, Vissick's followers would leave them alone or avoid them.

For the background, a hacker's lair/workshop or the streets (depending on who the player actually is) could work. In the images above, I don't like how intrusive the terminal is. I think it would be better if it was moved more to the background, where it would be a part of the image rather than stand in front of it, so you can still clearly see it, but without it taking away from the rest of the image. Making the whole image look more like scenery, with some elements catching your eye, but not one thing sticking out too much on its own.

**Reply by Moburma** [reply]
> Is there a possibility that the player isn't just someone controlling Vissick's minions at his behest, but rather one of the Unguided on the field doing the actual work themselves? In that sense you'd take the role of a different person for each mission, and the Unguided do have a free will after all, unlike Syndicate/Church agents that are basically mind-controlled puppets that only exist to serve their masters much like robots.

Yes, this is pretty much exactly what I imagined. Basically separate autonomous cells only loosely related with only really the dislike of The Man (and zealots) to tie them together. As the game would go on there would be more coherent solidified groups, such as in the other campaigns you get mentions of actual unguided leader characters. Even then I would imagine they only lead their corner of the world rather then represent everyone. So the player would not be one specific character, but an archetype of people acting all over the world.

I imagined it as early on he would just straight up broadcast public pronouncements to do things that everyone (including Eurocorp etc) could see, but as they are in disarray they can't counter easily so it doesn't matter. Then as things begin to settle he starts to use more covert messages to contact certain or more specific groups. Even then it would be more "anyone who is at x should do y as it stops Eurocorp etc from doing z". I also had an idea maybe some of the briefings would be back and forth conversations via email/chat with other members who disagree/have their own ideas etc.

I did some more AI generations, this is more the lab/garage angle @mefistotelis  and yourself mention, I came round to that more. I was going more for the person in the street on the way to cause mischief, but I think you're right, something like this makes more sense:

https://postimg.cc/gXxTz79x
![](https://i.postimg.cc/V6gQRyDW/20250513-2329-Cyberpunk-Lab-Scene-simple-compose-01jv5w67saf3fvgafgenknztz8.png)

**Reply by mefistotelis** [reply]
I like this one, having bikes there makes sense though not sure if they should have wheels..
The desk in front makes it similar to other backgrounds, and would make it easier to integrate some kind of holo projector.
UZI fits the scheme well. The electronics doesn't look like SW era.. though hard to say how the proper electronics should look like, maybe would be easier to just put a dismantled robotic arm or leg there.
While the game doesn't have any garages, I think the theme of a garage fits well.

**Reply by Moburma** [reply]
Yeah, I wouldn't take the images too literally at the moment, I'm going for the general feel/composition first, and then detail later. As it's crappy AI there is dumb stuff like the uzi being out of proportion and the circuit boards look weird and bendy. I will try and refine what AI can generate to get something more accurate, but I will ask around and see if anyone can actually properly create a 3D scene based on these ideas. A friend of mine is a graphic designer, and while I don't think he'd be that interested in doing a whole picture I could probably get him to do edits on a "close enough" scene to change details like the projector etc..

**Comment by Moburma** [comment]
Here's probably my best AI generated background so far:
![](https://i.ibb.co/213gS4dW/20250603-1109-Cyberpunk-Tech-Garage-simple-compose-01jwtm5sq4f59a09jc53zabaty.png)

Not perfect, but beggar's can't be choosers. I got it going in-game, but it shows up some obvious conventions colourwise they used - the middle needs to ideally be fairly dark for better contrast with the mission briefing overlays. The bottom also ideally should be dark as the icons are overlaid there.
